sdk/storage/Azure.Storage.Blobs/tests/ContainerClientTests.cs

```diff
@@ -2367,7 +2367,7 @@ public async Task ListBlobsHierarchySegmentAsync_Error()
                 e => Assert.AreEqual("ContainerNotFound", e.ErrorCode));
         }
 
-        [PlaybackOnly("Object Replication policies is only enabled on certain storage accounts")]
+        [Ignore("Object Replication policies is only enabled on certain storage accounts")]
```

Comment:
what's the plan to bring back this test?

Reply:
See above.